_posts/2018-11-15-Rust-survey-2018.md

```diff
@@ -0,0 +1,168 @@
+
+## **Feeling welcome**
+
+![Do you feel welcome in the Rust community](/images/2018-11-RustSurvey/23-Do_you_feel_welcome.png)
+
+An important part of the Rust community efforts are ensuring that the Rust project is a welcoming place for its users. New users should feel encouraged to explore, share ideas, and generally be themselves.
+
+When asked, both current Rust users and non-users largely felt welcome, though over a quarter of responses weren't sure.  There was also some regional variation in these responses.  For example, responses on the Russian version of the survey showed double the percent of unwelcome feelings at 4%. Mainland China showed even more at 8%.
+
+There's a challenge here to help Rust communities worldwide feel like they are part of what makes Rust unique, as Rust continues to grow a strong presence in more areas of the world.
+
+![Are you underrepresented in tech](/images/2018-11-RustSurvey/24-Underrepresented.png)
+
+Underrepresented groups continue to improve slowly year-over-year. The survey also highlights some challenges, as the number of women is still lower than the industry average of women in programming fields.
+
+# Rust Non-Users
+
+A big part of a welcoming Rust community is reaching out to non-users as well. As we have in years past, we again asked the reasons why people weren't using Rust. 
+
+![How long before you stopped](/images/2018-11-RustSurvey/2-How_long_before_you_stopped.png)
+
+For those who stopped using Rust, just over 50% stopped using Rust in less than a month. Likewise, roughly 50% of people who left Rust managed to use it for more than a month before stopping.
+
+![Why are you not using Rust](/images/2018-11-RustSurvey/3-Why_not_using_Rust.png)
+
+Many non-users responded that they did want to learn Rust, but there were factors that slowed them down. First among these is that the companies the responders work for do not themselves use Rust. Nearly one half of the non-users were blocked by the lack of company support. 
+
+Additionally, 1 in 4 non-users were slowed by the feeling of Rust being too intimidating or complicated. The work towards improving Rust IDE support has helped (down from 25% to 16%), though we still see a strong push towards even better IDE support among non-users.
+
+# Challenges
+
+As we've done in past years, we asked for your comments in where Rust can improve. This year, we see some familiar themes as well as some new ones in this feedback. The top ten themes this year are:
+
+1. the need for better library support
+2. a more improved IDE experience
+3. the need for broader adoption of Rust generally
+4. a richer ecosystem of tools and support
+5. an improved learning curve
+6. the need for important language features and crates to stable and supported
```

Comment:
"the need for important language features and crates to stable and supported" sounds like it's missing a word somewhere? Maybe "to **be** stable and supported"?